### PR TITLE
Blog app - setup and controllers

### PR DESCRIPTION
**Pull request by nevisende**
-  [X] Created a new app.
-  [X] Initialized the project with Git.
- [X] The project has a Postgres database set up.
-  [X] My app handles the URLs used the wireframes in the description:  
-  [X] For each URL I've created:
    - A route.
    - An action in the correct controller.
    - A view file.

-  [X] View files include only placeholders written in plain HTML, e.g:
    - The list of posts for a given user

-  [X] I added code to handle **only** the 4 required URLs.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Rails.application.routes.draw do
+  get 'posts/index'
+  get 'posts/show'
```
Comment (arslanbisharat) [inline config/routes.rb:3]: - [ ] You can make sure both user and post yield URL that correspond to the [screenshot](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/routes.png?token=ABIFLA7VAPBDIQKIFUV6II3BETCIO) in the [project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project.md) page

```ruby
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @nevisende, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You have added a descriptive Readme file. Good Job. ✔️
- There is no linter error is present. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (nevisende) [reply config/routes.rb:3]: Thank you for your review and time :)

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @nevisende 👋🏻, 

Good job making the requested changes!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

##Optional Changes

- [ ] Kindly add the `node_modules` folder to `.gitignore`.

Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by nevisende** [comment]
> 

Thank you for your review :)

**Pull request closed without merging**